text.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <main>
+        <header class="header">
+            <div class="logo"><a href="#">Jo</a></div>
+            <div></div>
+            <div class="menu">
+                <nav class="nav">
+                    <div ></div>
+                    <div ></div>
+                    <div ></div>
+                </nav>
+                
+            </div>
+        </header>
+        <section class="headline">
+            <div></div>
+            <div class="greet">
+                <h1>I'm Jonathan Glad to see you!</h1>
+            </div> 
+            <div class="presentation">
+                <p>I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+            </div>
+            <div class="connexion"><p>LET'S CONNECT<p></div>
```

Comment:
- [ ] As mentioned by the reviewers your design should stick as much as possible to the Figma design (eg: color, font,...). First your `LET'S CONNECT` button is in the `<button>` or `<a>` tag is not a paragraph, and also make sure your button has the same color as in the Figma design, please kindly fix that 🙃 🙃 

Your design

<img width="309" alt="Screen Shot 2022-09-22 at 11 53 25" src="https://user-images.githubusercontent.com/74501523/191718081-60b3be9b-4080-4eb4-b752-54bbc28d793c.png">

Figma design

<img width="309" alt="Screen Shot 2022-09-22 at 11 53 40" src="https://user-images.githubusercontent.com/74501523/191718120-e0bfffe1-87b6-407d-9380-73b5a8e1f3ca.png">